packages/loader/container-loader/src/container.ts

```diff
@@ -429,7 +429,11 @@ export class Container
 								// Depending where error happens, we can be attempting to connect to web socket
 								// and continuously retrying (consider offline mode)
 								// Host has no container to close, so it's prudent to do it here
+								// Note: We could only dispose the container instead of just close but that would
+								// the telemetry where users sometimes search for ContainerClose event to look
+								// for load failures. So not removing this at this time.
 								container.close(err);
+								container.dispose(err);
```

Comment:
Does this line run even if the "closed" event fires before we call dispose? (i.e. does the `reject(...)` call above stop all other stacks in the `.load(...).then(...)` portion)

Reply:
Yes, it will continue as promise reject will not throw. Since we are not doing **return reject()**, the execution will continue and the caller who is awaiting will get a rejected promise.